packages/calling/calling/tsconfig.json

```diff
@@ -65,6 +65,7 @@
     // "experimentalDecorators": true, /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true, /* Enables experimental support for emitting type metadata for decorators. */
     /* Advanced Options */
+    "stripInternal": true,
```

Comment:
What was the reason for adding this?

Reply:
To fix the import with media-helper, it was added. Not needed anymore